LoopKitTests/TherapySettingsTests.swift

```diff
@@ -295,18 +295,18 @@ class TherapySettingsTests: XCTestCase {
     """
     
     func testInsulinModelEncoding() throws {
```

Comment:
Github won't let me comment on the line for some reason, but line 242 needs to be updated to `"exponential" : "rapidActingAdult"` so the encoding/decoding tests pass